docs/source/run/parameters.rst

```diff
@@ -59,9 +59,22 @@ General parameters
     Maximum number of time steps. `0` means that the 0th time step will be calculated, which are the
     fields of the initial beams.
 
+* ``hipace.max_time`` (`float`) optional (default `infinity`)
+    Maximum physical time of the simulation. If ``t + dt > hipace.max_time``, the time step ``dt``
+    will be reduced so that ``t + dt = max_time``, both for the adaptive and a
+    fixed time step.
```

Comment:
Minor rewording
```suggestion
    Maximum physical time of the simulation. The ``dt`` of the last time step may be reduced so that ``t + dt = max_time``, both for the adaptive and a fixed time step.
```